lib/configs.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   accessToken: '31964eb773659fb3aad6e982c692d2e09ff4b403',
-  regex: /^[a-zA-Z]+-\d+:.*/,
+  regex: /^[a-zA-Z\d]+-\d+.*:.*/,
```

Comment:
do we want to enforce people to add comment for the Story? Now something like "story-123:" will pass as pr message.
/^[a-zA-Z\d]+-\d+.*:.+/